scaleapi/api.py

```diff
@@ -109,9 +109,28 @@ def _api_request(
         json = None
         if res.status_code == 200:
             json = res.json()
+        elif res.status_code == 409:
+            retry_history = res.raw.retries.history
+            # Example RequestHistory tuple
+            # RequestHistory(method='POST',
+            #   url='/v1/task/imageannotation',
+            #   error=None,
+            #   status=409,
+            #   redirect_location=None)
+            if retry_history != ():
+                # See if the first retry was a 500 error
+                if retry_history[0][3] == 500:
+                    uuid = body["unique_id"]
+                    newUrl = f"https://api.scale.com/v1/tasks?unique_id={uuid}"
```

Comment:
```suggestion
                    newUrl = f"{self.base_api_url}/tasks?unique_id={uuid}"
```

Reply:
Probably would be ideal later if we have an API method that directly returns a single task for the given unique_id (similar to getting a task with task id)

Reply:
yeah that would be ideal